chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# stop on first error
+set -e
+
+# validate
+if [ "$#" -lt 2 ]; then
+  echo "The script requires at least 2 arguments - filenames"
+  exit 1
+fi
+
+# iterate thru files
+while (( $# )); do
+  file=$(basename $1)
```

Comment:
It seems that because of this, files would be searched in root folder only, i.e. files inside other folders won't be found

Reply:
Fixed, according to task "it should create these files in a local directory"